src/auth/registration/BrowserForm.tsx

```diff
@@ -0,0 +1,71 @@
+import { FormikTextField } from "form/FormikHelpers";
+import { MarketingOptIn } from "auth/registration/MarketingOptIn";
+import { useField } from "formik";
+import { msg } from "@lingui/macro";
+import { useLingui } from "@lingui/react";
+import { MarketingChannel } from "auth/registration/MarketingChannel";
+import { Box } from "@mui/material";
+
+export function BrowserForm() {
```

Comment:
nit: it looks really ugly when the whole form moves when validation errors appear.

Please can we reserve space for the validation error so it just appears rather than causing the entire form to reflow?